printing/paper

```diff
@@ -87,6 +88,11 @@ def main(argv=None):
     parser_refund.add_argument('--reason', '-r', required=True, type=str)
     parser_refund.add_argument('user', type=str)
 
+    parser_forward = subparsers.add_parser('forward', help="allow a user to print above today's quota")
+    parser_forward.add_argument('--pages', '-p', required=True, type=int)
+    parser_forward.add_argument('--reason', '-r', required=True, type=str)
```

Comment:
Forward probably in itself is the reason, I don't know if this needs to be a required field. What do you think?

Reply:
I think this is still fine, and it's a database constraint anyways

Reply:
Despite the reason being clear, I think this would be important to include due to the human aspect of it. A person who is issuing the forward request will need to think twice before actually increasing the daily quota for a particular user.